lib/api/update/photo.ts

```diff
@@ -6,7 +6,8 @@ import smartcrop from 'smartcrop-sharp';
 import to from 'await-to-js';
 import { v4 as uuid } from 'uuid';
 
-import { Account, Constructor } from 'lib/model';
+import { Account } from 'lib/model/account';
+import { Constructor } from 'lib/model/resource';
 import { APIError } from 'lib/api/error';
```

Comment:
Make sure to sort these imports alphabetically; the `APIError` one should go before the `Constructor` one.